org.codecoop.kjo.bookletimposer.appdata.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>org.codecoop.kjo.bookletimposer</id>
+  <name>Booklet Imposer</name>
+  <metadata_license>GPL-3.0-or-later</metadata_license>
+  <project_license>GPL-3.0-or-later</project_license>
+  <developer_name>Kjo</developer_name>
+  <summary>Booklet Imposer app</summary>
+  <description>
+    <p>
+      Achieve some basic imposition on PDF documents
+    </p>
+    <p>
+      NOTE: This flatpak is not verified by, affiliated with, or supported by the developers of Booklet Imposer.
```

Comment:
I don't think you need such disclaimer for open source application.

Reply:
Should i remove it then ?

Reply:
Yes.